resources/lib/modules/bluetooth.py

```diff
@@ -73,48 +76,26 @@ def exit(self):
     # ###################################################################
 
     @log.log_function()
-    def init_adapter(self):
-        dbusBluezManager = dbus.Interface(oe.dbusSystemBus.get_object('org.bluez', '/'), 'org.freedesktop.DBus.ObjectManager')
-        dbusManagedObjects = dbusBluezManager.GetManagedObjects()
-        for (path, ifaces) in dbusManagedObjects.items():
-            self.dbusBluezAdapter = ifaces.get('org.bluez.Adapter1')
-            if self.dbusBluezAdapter != None:
-                self.dbusBluezAdapter = dbus.Interface(oe.dbusSystemBus.get_object('org.bluez', path), 'org.bluez.Adapter1')
-                break
-        dbusManagedObjects = None
-        dbusBluezManager = None
-        if self.dbusBluezAdapter != None:
-            self.adapter_powered(self.dbusBluezAdapter, 1)
-
-    @log.log_function()
-    def adapter_powered(self, adapter, state=1):
-        if int(self.adapter_info(self.dbusBluezAdapter, 'Powered')) != state:
-            oe.dbg_log('bluetooth::adapter_powered', 'set state (' + str(state) + ')', oe.LOGDEBUG)
-            adapter_interface = dbus.Interface(oe.dbusSystemBus.get_object('org.bluez', adapter.object_path),
-                                               'org.freedesktop.DBus.Properties')
-            adapter_interface.Set('org.bluez.Adapter1', 'Alias', dbus.String(os.environ.get('HOSTNAME', 'libreelec')))
-            adapter_interface.Set('org.bluez.Adapter1', 'Powered', dbus.Boolean(state))
-            adapter_interface = None
+    def find_adapter(self):
+        self.dbusBluezAdapter = dbus_bluez.find_adapter()
+        if self.dbusBluezAdapter:
+            self.init_adapter()
 
     @log.log_function()
-    def adapter_info(self, adapter, name):
-        adapter_interface = dbus.Interface(oe.dbusSystemBus.get_object('org.bluez', adapter.object_path),
-                                           'org.freedesktop.DBus.Properties')
-        res = adapter_interface.Get('org.bluez.Adapter1', name)
-        adapter_interface = None
-        oe.dbg_log('bluetooth::adapter_info', 'exit_function', oe.LOGDEBUG)
-        return res
+    def init_adapter(self):
+        dbus_bluez.adapter_set_alias(self.dbusBluezAdapter, 'LibreELEC')
```

Comment:
hardcoded LibreELEC should be replaced with @@DISTRONAME@@ or something we can sed at build time to support downstream..